opennms-base-assembly/src/main/filtered/etc/opennms-datasources.xml

```diff
@@ -4,32 +4,32 @@
   xsi:schemaLocation="http://xmlns.opennms.org/xsd/config/opennms-datasources 
   http://www.opennms.org/xsd/config/opennms-datasources.xsd ">
 
-  <connection-pool idleTimeout="600"
-    loginTimeout="3"
-    minPool="25"
-    maxPool="50"
-    maxSize="50" />
+  <connection-pool idleTimeout="\${env:OPENNMS_DATABASE_CONNECTION_IDLETIMEOUT|600}"
```

Comment:
Do we really need these escape sequence  infront of all metadata ?

Reply:
Removed the escaping in a separate commit; it was just to avoid unintended variable expansion from Maven. No strong objection on this from me.